app/views/pages/home.html.erb

```diff
@@ -1,71 +1,136 @@
+        </ul>
+      </div>
+      <% end %>
+      <% if @organization.provider? %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.jobs.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% if @organization.default_stream.active_jobs? %>
+          <% @organization.default_stream.job_tracker_status_groups.each do |status, jobs| %>
+          <% if jobs.any? %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.job_status_group[status].icon_class, class: "pod-icon pod-job-tracker-status #{status} me-2")%>
+            <%= t('.organization.jobs.status_html', count: jobs.count, status: t("job_tracker.status_group.#{status}").downcase) %>
           </li>
-          <li class="list-group-item border-0">
-            <%= link_to 'More documentation (wiki)', 'https://github.com/pod4lib/aggregator/wiki', class: 'text-muted' %>
+          <% end %>
+          <% end %>
+          <% else %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.job_status_group['completed'].icon_class, class: "pod-icon pod-job-tracker-status completed me-2")%>
+            <%= t('.organization.jobs.status_html.zero') %>
           </li>
+          <% end %>
+          <li class="my-3"><%= link_to t('.organization.jobs.all'), organization_path(@organization) %></li>
         </ul>
       </div>
-    </div>
-    <div class="col pt-3 px-3 pt-md-5 px-md-5 text-center overflow-hidden">
-      <% if current_user %>
-        <div class="my-3 p-3">
-          <h2 class="display-5">My organizations</h2>
-          <ul class="list-group lead align-left">
-            <% organizations = Organization.with_roles([:member, :owner], current_user) %>
-
-            <% organizations.each do |org| %>
-              <li class="list-group-item border-0">
-                <%= link_to org.name, org, class: 'text-muted' %>
-              </li>
-            <% end %>
-          </ul>
-        </div>
       <% end %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.actions.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% if @organization.provider? %>
+          <li class="my-3"><%= link_to t('.organization.actions.upload_file'), new_organization_upload_path(@organization) %></li>
+          <li class="my-3"><%= link_to t('.organization.actions.view_default_stream'), organization_stream_path(@organization, @organization.default_stream) %></li>
+          <% else %>
+          <li class="my-3"><%= link_to t('.organization.actions.harvest_data'), organization_path(@organization) %></li>
+          <li class="my-3"><%= link_to t('.organization.actions.data_docs'), 'https://github.com/pod4lib/aggregator/wiki/Harvesting-using-ResourceSync' %></li>
+          <% end %>
+          <% if can? :manage, @organization %>
+          <li class="my-3"><%= link_to t('.organization.actions.edit_org'), edit_organization_path(@organization) %></li>
```

Comment:
@thatbudakguy I pulled the branch locally and was clicking around, noticed with the now-merged changes to the Manage Org page, the `edit_organization_path` no longer exists, the good link would be: `organization_users_path(@organization)` . I wonder if we should add tests to make sure the homepage links are all valid? Or maybe this is just a one-time thing since  a lot of simultaneous changes are happening.

Reply:
oh yeah when i was writing that it didn't exist, good catch. i can update these now!

Reply:
fyi I updated this one, but i think one or two of the others link to tabs on the organization page, and are thus blocked on #531